portfolio/src/main/webapp/index.html

```diff
@@ -16,9 +16,13 @@ <h1>Betty's Portfolio</h1>
       <p></p>
       <button onclick="changeBackgroundColor('lavender');">My favorite color</button>
       <p><p/>
-      <button onclick="getMessages()">Hello World!</button>
-      <div id="messages-container">
-          <ul id="messages-list"></ul>
+      <button onclick="getComments()">See Comments</button>
```

Comment:
can we just show the comments on load? it's a bit weird having to click to see the comments?